pinot-tools/src/main/java/com/linkedin/pinot/tools/perf/QueryRunner.java

```diff
@@ -673,6 +692,44 @@ public void run() {
     }
   }
 
+  @ThreadSafe
+  private static class Statistics {
```

Comment:
Not required for this PR, but wouldn't a better abstraction be to support multiple columns in this class, so that the client code does not have a list of Statistics?